src/Workspaces/Core/Portable/Shared/Utilities/AsyncBatchingWorkQueue.cs

```diff
@@ -157,6 +160,7 @@ private void TryKickOffNextBatchTask()
                     _ => ProcessNextBatchAsync(_cancellationToken),
                     _cancellationToken,
                     (int)_delay.TotalMilliseconds,
+                    _asyncListener,
```

Comment:
oops!

Reply:
This wasn't a feature previously, but I added support for expediting this queue due to its importance in these tests.